Doc/c-api/mapping.rst

```diff
@@ -50,20 +50,20 @@ Mapping Protocol
 
 .. c:function:: PyObject* PyMapping_Keys(PyObject *o)
 
-   On success, return a list or tuple of the keys in object *o*.  On failure,
-   return *NULL*.
+   On success, return a list of the keys in object *o*.  On failure, return
+   *NULL*.
 
```

Comment:
Add a "versionchanged" directive.

This is needed for users that write a code that can be used with older Python.